tests/scripts/generate_psa_tests.py

```diff
@@ -35,6 +35,14 @@
 
 def psa_want_symbol(name: str) -> str:
     """Return the PSA_WANT_xxx symbol associated with a PSA crypto feature."""
+    # PSA_WANT_KEY_TYPE_[RSA/ECC]_KEY_PAIR symbols are deprecated and they should
+    # be replaced soon with newer PSA_WANT_KEY_TYPE_[RSA/ECC]_KEY_PAIR_yyy in
+    # library's code and tests. Until this happen though, they have been
```

Comment:
Do we have an issue to track the update of the python scripts?

Reply:
If you mean the issue for when this fix will be removed then the answer is no, we don't have it yet. 
I think this can be done in the follow up issue/PR that I mentioned [here](https://github.com/Mbed-TLS/mbedtls/pull/7641#discussion_r1205775058).

Reply:
I've just created the issue for reverting this change in the future: https://github.com/Mbed-TLS/mbedtls/issues/7673